test/fs/test_fs_js_api.c

```diff
@@ -52,14 +52,14 @@ EM_JS(void, test_fs_readFile, (), {
 
 // createPath should succeed when called on existing paths ( https://github.com/emscripten-core/emscripten/issues/23602 )
 EM_JS(void, test_fs_createPath, (), {
-    FS.createPath('/', 'home', true, true);
-    FS.createPath('/', 'home', true, true); // Creating an already existing path
-    FS.createPath('/home', 'nested1', true, true);
-    FS.createPath('/home', 'nested2', true, true);
-    FS.writeFile('/home/nested1/test.txt', 'a=1\nb=2\n');
-    FS.writeFile('/home/nested2/test.txt', 'a=2\nb=4\n');
-    var read1 = FS.readFile('/home/nested1/test.txt', {encoding:'utf8'});
-    var read2 = FS.readFile('/home/nested2/test.txt', {encoding:'utf8'});
+    FS.createPath('.', 'home', true, true);
+    FS.createPath('.', 'home', true, true); // Creating an already existing path
+    FS.createPath('home', 'nested1', true, true);
+    FS.createPath('home', 'nested2', true, true);
+    FS.writeFile('home/nested1/test.txt', 'a=1\nb=2\n');
+    FS.writeFile('home/nested2/test.txt', 'a=2\nb=4\n');
+    var read1 = FS.readFile('home/nested1/test.txt', {encoding:'utf8'});
+    var read2 = FS.readFile('home/nested2/test.txt', {encoding:'utf8'});
```

Comment:
Can we still test the absolute paths as well? Those should still work IIUC.

Reply:
There are several confugurations where absolute paths don't work.   I suppose we can ifdef them here.  Let me see.

Reply:
Another option is if you see absolute paths are checked sufficiently in another test.

Reply:
Actually do you mean you want me to duplicate all the tests here somehow?

Reply:
OK, I worked around the issue by skipping the abs path tests under NODERAWFS for now.